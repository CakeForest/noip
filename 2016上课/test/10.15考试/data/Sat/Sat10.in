1000
86090694 89352275 4
79157074 17397200 49949649
56356096 26347064 282945
21863292 97103286 34013298
81670815 1 34171016
55409340 70294620 52568142
76633245 98908675 23838436
95702299 20958128 60596272
33421968 42031715 26259235
23925428 46002955 44818470
79771904 2055456 66221024
5562480 37989210 13841460
96502876 82285840 93849376
76980504 13430480 56439654
32147675 52668934 22474600
20615598 49537885 25090077
94823908 15953685 91179750
81438218 40198000 2518880
74489415 73118416 47724720
40533500 48895935 78527818
3506860 17397952 96743660
35775680 68127375 14734357
144985 16185966 29716466
13955497 29126283 32656775
3873980 18559520 40454924
83130786 57979141 30108430
36040264 78743170 13690944
57849068 22405936 23974656
96621336 25414713 58185216
64626798 11383307 26944704
86003436 34793625 49516320
1319146 30082878 17106358
13149358 26702035 97007735
7642107 16611110 59849150
55176001 18019008 57232462
59833764 6188886 25258790
88230519 64026664 78916708
52291260 33973716 91542150
37418289 7979840 90734644
17705063 69370948 90962915
49266960 70193627 55874356
1151608 68037540 60276352
52734580 44117458 12704193
42776838 80863780 6248360
16206088 1367693 27541694
34201478 47802685 60558915
72139820 1474354 4772500
10085935 45329208 25262080
77483989 26107172 39737612
74814670 87626296 35245504
46535522 67033160 95240296
72156854 3787980 16472014
12310520 31369296 34405158
3311016 53422255 42124232
2848106 89268114 20053603
87269030 18251060 88150644
89823356 52842259 26212254
19244636 15795552 20263122
43552500 54293074 3686958
76676585 25225734 6953558
98903604 95265252 67065270
41596120 96236012 54441757
76534881 16064724 47450732
20602315 3493204 3957756
49599626 8779296 86194552
22396448 10424107 80184925
36809741 3509184 4463578
38885646 47935060 317080
12258958 69943945 49511502
16809900 89422083 47972136
40364554 93722800 12747912
20930416 15145632 87329936
88410680 8882080 74476345
44845576 81995125 80932912
25247074 79316712 1912582
17958550 79737654 10806272
43660760 24372306 40139853
97599695 57169030 6444980
51534140 20886752 62143872
79651388 42820746 61205296
58721750 94497624 2537061
47766624 91616070 76426878
30607748 73868984 46625594
59456322 5142599 42532230
78264107 4949406 5303456
99968804 33380200 38900544
25593081 50969412 91789070
76481740 58441923 92289266
84353617 453543 39989002
59894843 51273580 49266190
75090629 59030200 14285423
24561800 48661360 39804134
66192610 94355105 27109367
14531802 67593285 97530752
7990374 28465772 5812215
24084488 55546202 25189450
77969650 30856910 17940144
84832128 49969600 56903512
58355560 10044928 20103830
4030130 9875185 3609864
72111284 20921472 1380480
37651283 40430440 65348997
18164594 99505816 25891786
22189244 6240916 41639624
22663914 22148824 51885514
52730660 81389920 9264904
5740396 76221358 12468969
12799680 10519248 24137302
97599544 24104265 71936508
78180072 15863652 7154127
42486506 22188738 11473136
55980599 40549200 69312999
17440645 60526898 10939602
37357564 78955758 35378780
16949530 10095192 273552
41847264 93155902 89041280
94327150 55906576 38246580
44607448 78358816 1710880
6219376 59948192 46077549
75620753 10313780 11440009
26131668 40323150 6050288
38349896 22826496 4443016
51077640 54108465 1269324
99238102 64275801 31268950
55408286 41742202 69880818
60587678 20004218 59483712
70440898 74014838 79052083
26046618 77020554 2151122
90293455 74109166 44070628
7029230 5908160 69837542
33356658 29173320 14020030
35435120 5294327 16710503
6270791 72192812 73596360
33711530 40138208 84434034
52596980 86648227 75967820
29416283 68455500 21500890
64675411 47133754 95720096
54675968 66921600 4465356
58696276 2669244 25715170
45959210 13285640 65224740
78264357 42376342 61441256
20597334 44427366 46137805
15385920 54719282 43662155
80183214 57072616 14780025
22027202 21507417 88983642
59982061 24434147 36920836
64957435 41020208 64949849
81330788 70713747 19696125
1926776 39437586 58693102
50393510 27441584 88273895
53942296 15695077 51038090
4270850 81479402 99131760
45313690 48910528 8609920
50533879 44613252 14842960
89879543 46816730 10831946
88752882 49194584 46213232
88464962 59510142 86269449
50976718 1341028 30897431
83943726 60866906 18248630
6176816 14699100 58868997
67740772 20570110 20427270
92093820 18408300 54859972
11905686 23207682 89593968
38124639 25184964 71466870
61060254 23519307 4908350
28224795 36236452 2586208
61941549 22700560 55841328
58038816 52616256 19172064
90867040 81950892 75717665
45221816 77410851 16010392
46447796 59758965 40870349
16447463 51960204 1371896
69296572 75050700 7761930
49882917 93254882 39595297
58430556 25956739 94206794
61054715 61851940 64949477
43672778 77967802 87423550
36443408 57836469 49038218
28921394 48307520 18698356
74993480 6354414 68435418
35759208 68518824 67496800
54240372 17986462 62682350
97851177 60648387 29957688
52062710 31624648 59153365
64620564 95370664 6269120
877794 43338461 74316302
84562812 80858908 31332364
69345205 63950030 73629310
97040 19507860 89768950
81790206 61966670 30016233
78633894 79036082 92771239
14744205 33932710 42116367
94413776 47310261 17698134
9489280 95723599 72785784
4645532 61656302 35124892
69179106 94326568 17566244
22870488 64842717 45741325
27319672 79572200 11793200
51360932 85799864 66611516
33047584 9004420 28687322
58389615 49155675 11075328
6716088 78395235 55199830
78000067 97814970 45703299
83602599 96730563 10373072
12254424 14706720 56397495
30458540 3619442 69793036
7436185 4335209 81531560
34969898 55965855 37700560
18331701 35801910 23408865
46066815 90917209 66661250
1718982 58804190 6975510
71795736 87735487 30056360
74792379 40203891 84387963
93458336 60535188 37601120
28353806 12022383 81710808
55331551 81048800 28774356
45548458 36117664 54156930
76911871 7608640 97613585
76974110 63258452 38754980
47450648 67768646 17635136
56178713 50423741 46107464
25810660 16823080 84547816
27058166 36137675 73465005
19505200 452333 28277664
47979920 68393414 82054087
94620491 3597568 53967632
18326760 60412923 34746536
10873912 11783555 30220800
1290912 3994905 12428944
5549491 33548330 94281673
90642681 13665400 26680958
64078136 93505784 32183950
84895520 90695284 98816268
56681824 8614155 9224320
8741970 66889056 7477672
33059928 73975140 11458312
16460240 63030960 18727267
14617218 81676650 35330120
56304525 99404918 66387488
58433332 90836488 62741776
25986504 34852244 94869150
68851408 61156318 64334628
48733254 35544644 77120762
22644630 34746106 220820
57055632 41522449 3670776
97084078 54911064 82949200
31741088 3930480 18066598
4368891 17133824 8872816
1061330 68574294 28175350
2779600 75922511 3338008
91783160 77342405 83823290
67554680 90089813 76893870
14525035 71262791 16877500
1883864 4472992 63343600
68057440 35270532 8901468
71742664 29665435 99491004
89952328 35489704 91271641
69891280 68636944 14676287
81933669 37746000 14685848
1431208 61231700 22207275
55953246 41181072 20335768
17156675 47259345 17019684
67038436 4899910 35334216
52348355 77842286 3925416
2818413 5588985 10634015
97468272 69958230 98790608
14134076 83079715 60352270
72532040 42472156 55267542
34899416 55158585 92217520
75323060 89591306 44147180
9083024 83613215 34211100
62841395 30132266 31483650
97341491 77186469 67222173
93307168 5839565 82998700
82335855 13341498 6590056
23543068 75615630 52888316
20544294 66904550 55949250
63467252 3462620 24521428
53543990 24427862 34861550
26750792 34232744 25126452
69648100 19693912 24988606
29321452 91846436 14004900
28922866 41144976 7904960
14405030 47736334 6826336
41843335 11945874 23159140
77246825 55303034 88877401
41089934 51874183 67228820
23316357 43901775 18705075
21233025 45021448 75709688
19332145 5922080 43100160
24600146 62280536 41823848
4271174 68190412 66380625
18357300 28783390 38305576
22144951 3801046 36154150
37729865 77918724 3989936
89921808 53881972 11677873
59134028 47268496 71887325
24827139 40434170 17308520
74724739 91641610 2058220
73291452 99703303 11163140
60464896 9973640 32896044
64714536 88281616 95196564
91661990 88964184 15419032
9886919 95026065 4974460
49899551 36838230 10231760
33648000 42501750 40336704
26611856 41615699 27007175
79303650 54966875 42620782
2617824 45290304 64293125
95958926 31230550 49151676
57848400 91965820 55825538
66695950 76310496 57066032
25728002 71207252 20199973
95846376 45001324 82058560
52810273 37287810 74836979
87330617 20663465 10193220
51153960 65132052 98597970
84061970 25542881 84315854
71985720 32593650 8991300
99139890 8304872 45206572
79840800 22424659 87414410
64617660 67683 29143684
21328200 36828960 27506378
33221805 82858210 23206054
46486561 15303848 78433984
31233812 69856985 95040718
79888028 74692716 27598475
45218904 97567182 59571908
63548250 25897296 99716132
26672085 50717700 6066656
28946800 51605631 94560378
63664940 26534052 98928845
88180050 65160270 55098880
12165384 7205605 35746984
57357200 12035460 25047270
17870306 32691940 72657992
75308580 98982176 42892125
30148696 62583902 82455164
59234647 44992045 9525000
98048170 30674794 24741354
63741932 91004368 1313491
83290000 78270800 21690595
66007872 3338994 68864382
70405020 82634536 4993461
2777328 20853568 4964856
31055940 34538408 79163485
46391031 17495030 90055586
77858310 10938274 58450588
58735522 52171560 98045355
71739832 7054864 75651975
68160944 1746752 62626888
20582128 54404872 95711556
57783879 54138620 81379705
64428425 2073561 57233936
30392112 32947698 57250800
27122740 23403190 98280540
15575863 37743042 12300132
94219835 19081582 96044885
62775725 78780518 74477352
98224100 12982992 77010749
89062659 37169376 81020948
305505 401788 58959020
48469880 89499122 74896910
54684413 7946400 25055350
21144924 18118080 27983153
30263508 2443964 96012862
21139608 28718500 38067136
38940234 18802068 43769915
32021050 8366932 30650320
11370000 29502575 84118844
11325672 85430473 7530740
59734046 27166821 73993624
47275378 60762530 4844896
61134816 1816946 44516835
17666800 88805872 88256132
23430914 6761564 38607731
43132524 4632350 20456128
18137590 82323445 16448475
90053582 86932465 3234128
37622566 90657809 28283932
27199564 41306256 64895341
24849753 52491580 51902847
3455520 38793928 6434969
97487118 90911684 97003250
16252912 6606434 27620798
73538648 51146160 17169408
59058899 24370740 3687880
91010656 2709666 51279580
56837888 1247040 10628192
19378480 86955856 43507820
4623570 27642856 41865820
14962920 21630470 90198575
37076270 78024692 49806420
21120720 22055366 9225523
19978140 30387625 44555268
2719020 77537868 55707456
55865732 50930352 7050728
40912015 3122279 64965543
10729752 34325506 58624065
16774362 83596522 15759700
65128512 15729577 83920409
46585901 4589070 95777660
67964787 81280417 34589500
32846229 36979110 13160654
885172 32520024 42595264
1404915 20390480 37373624
65922154 28438164 59098318
64049600 38239798 7384239
67632442 24884700 81800704
54094173 50543702 93538464
54030625 19420908 54982935
24975200 46590100 84479680
53958928 82761330 25335305
79712960 65404900 12966918
39704438 4832080 99625732
75208047 38125229 9968512
43826180 34637106 52168370
13724540 16825779 59354125
67915248 3852773 23871498
32695010 49872621 69803023
50090238 6018440 39712680
41456835 64915808 96459736
41502918 73976270 41612725
58247016 80933120 2048245
34674395 52318536 23623710
80553110 29096781 37677780
1202306 56328580 14543360
67787256 69916400 75882731
87558284 49933806 39184820
5877802 99029512 49021091
34633765 22521840 30471107
67853444 35801224 39169885
69543040 82671520 94568912
29127712 26134372 37099020
11060676 9024808 9701949
54031692 78538048 48970520
60097400 62046735 99814215
24178885 57652368 67532586
37586720 6814824 3951486
6609227 49165183 39734140
74932628 59670000 86866150
93112028 27802736 49508765
65761338 58606320 71656702
53440965 4940605 96233872
29606639 63338457 52264250
16015094 17561724 22596522
34494472 9205500 46721020
61791228 45843961 52458840
1340352 23178768 2309427
8656537 79262647 40333120
12072802 59347026 91588260
56450107 83905600 26793425
83166869 67350400 11051168
38353233 16636280 79951098
75575785 79597172 81986196
92884664 52052758 674400
84924365 84670540 26680534
24428670 30209505 2691407
94596652 68011508 49849916
8674520 12783149 87917655
93462666 8486384 42078726
33005320 62573328 91787112
53928282 15692208 43432735
60445900 85311578 51613420
91074472 52657334 68616840
85061155 32662588 17225012
33326863 35855265 24133894
3140170 81565952 99424700
5923658 26705180 40899804
64541538 5532448 17765810
17183834 35625300 3251104
79844000 51818544 89658908
28788224 95628978 36768794
13619840 33323494 23329746
4647656 38982870 47660744
17499290 68086958 38746880
22136960 19156112 49786376
49467456 77087570 40599396
98557960 60441597 6880379
75235710 9894176 47694040
42506180 8785718 21313610
49385560 1217556 29820232
57632515 7326329 32168068
79934757 13019110 47017725
75460484 33619960 61067934
77032906 7716800 57183769
36984393 72703459 9894360
96074126 88637740 43023770
8797754 55660610 48491534
83850435 12509678 12530592
97854965 67105176 5933099
57559911 84983268 836712
8741324 48602532 25853384
48127176 13167792 3476660
15371676 14239125 96178883
88991812 80946170 36332808
87708480 42349578 83190792
82277100 1860320 48895210
48778040 81335062 11114896
5649987 86258976 15371912
34022400 53269415 37012360
3389720 76043095 10252034
89894008 36851299 91698464
36691110 49950950 77648900
40333300 9455259 71384411
83493459 74094720 73137272
13436859 47845861 13940190
38338080 72360470 37933432
36832288 43999563 45614105
12864477 2380664 68763090
80406912 30231079 79940280
50330647 85384946 46927940
50926717 75019591 79281998
83645960 60693706 2701776
3970132 31115088 54926304
26657881 65905315 10954792
28962906 61210760 87049581
33727254 412295 88987186
65672256 20250054 80545830
10616370 3951516 75745173
76651220 23295648 67641064
62496035 13923752 99670926
2567894 98262736 33134992
93366770 58090590 99968690
15124247 56415900 78081002
89744960 61995874 16547722
44992025 4732000 2165188
87415060 13124916 14501592
70607402 29186037 71958605
33909564 28042372 91904618
34432695 35847594 90014732
56622945 43770677 53658225
32618990 31118980 9003865
30681154 33631950 78252269
53714032 66779112 64862650
31160907 32730751 22893250
26693784 15315124 11820226
10568745 59196916 36801322
42469095 48016032 76676374
12735064 32826420 1433952
14822840 38363139 57255632
42817120 239040 25377012
6129674 49516494 1615100
97800060 8315000 26375040
41854720 70058375 29831172
46183068 36872000 50008783
48568432 95900604 14806734
882522 25797688 60935920
36667848 27768792 4963032
74908834 31494042 2177878
45776240 3102705 5268755
653958 43080970 44308
47075018 37840016 82739424
29239950 60181568 39125760
32757741 29440714 98068598
73352972 87467680 31628530
97416000 12253056 13001009
18265457 30020536 25259530
62061899 46312324 11815995
92541740 59346967 15968512
81963952 71279648 58006912
43315216 37477748 90494720
67638262 1350700 23610416
64489050 49050455 74720526
39590100 3530754 9059370
29814770 75982784 66884090
55342882 26129408 82859768
83280645 49078614 1910336
76343862 65968096 24955031
2276333 628403 44721234
7781222 23755916 69344080
68351625 81849646 93294634
56715425 50845824 37088510
72268150 91152116 37286128
52816200 39922690 44234650
48908193 15289365 56160906
3432813 49112896 50112832
50276772 13376592 1578750
30683974 24909640 79517472
80410698 28224646 73225965
88290218 73587834 62930376
86206152 60524550 67276684
95897470 18120533 64758952
11246005 2206219 19123391
42084995 71906416 71724187
14927563 10232950 79494908
2226150 18470757 18420837
57152316 16216855 19654362
86059625 80323992 67371325
11214333 35453590 83788760
72869120 199263 60749865
74899891 17069932 57519148
95541361 55444600 49632630
4852984 50493168 7899170
20701663 79602674 83071743
53512650 4545404 15427800
94687616 53164123 6496138
13772336 69237168 26490160
62793110 5306522 89652670
72868144 12674775 4926016
25698680 52443672 93836062
69225868 10751025 26782050
34516084 59398512 16034137
73375740 22490950 80098650
88644206 12116168 3006503
83539837 54527380 7024377
64408795 77408932 19054610
11571701 93622500 50544000
67539776 91586676 45430402
96837084 57998681 53777600
42464301 91472040 82133480
85442622 78750064 81198956
81211037 69602674 72299253
44864868 61153118 71325669
31821472 3762344 44166448
64407920 23588604 25527598
38636070 13103106 52693363
40546872 17185896 91386880
83435710 81790295 20667299
69417844 89958199 92860416
15616960 38178346 89245280
26949484 17387560 51780600
74266129 51394518 31519370
62499485 92623533 38452458
80262155 6048046 16477614
57997815 342072 73247690
95419470 87667304 8672813
89639225 44214008 54524553
64382280 55844048 23984538
24140833 52807458 60269200
74273712 68766774 17300082
34465306 26231401 43153840
51318308 69953332 84638258
21374508 16013192 58853610
18764835 67185329 91169780
70777990 26079208 51957626
46852020 19123090 56474085
9715412 1488721 43793134
20778270 21217505 91050000
28483936 95858936 29267424
49539721 89812688 24452736
53344583 59842336 61448368
20120168 99510144 56463088
95959626 39583016 58776176
2739744 72208808 11358054
74012684 57009748 21568062
15677770 70132032 56954118
89509477 87157760 28002080
30714280 97851456 12253180
5075230 45798292 68204385
73938620 44992380 40872407
89342872 85125108 45688615
34496128 58572006 12612146
76846228 13258816 18542784
34401900 50021710 29229032
1090329 5889378 55381720
47169895 27503865 57092320
25782101 15634088 55927245
70182347 85367907 71170680
23310144 16892193 25821056
70536040 15548998 44297510
38451576 8606976 84224035
25931045 24216445 23623192
30037826 91391404 66505006
44751984 61932117 11041501
83070980 33132940 25653323
1911008 79430591 47166001
95164838 92292460 22078484
75510792 9028575 68006769
33137635 28382736 59938928
64960569 26061964 23546239
75466978 27404523 48815920
7684432 73555328 2710864
10882024 77188959 66169414
56811076 42217303 56298032
46432440 85506386 95962660
4050471 62147806 26255986
67844146 9159788 72410720
62047078 1758000 51004598
20843670 44066816 74959802
84222864 77320868 13015803
6816947 9953384 1272158
35236032 53249613 28286734
86044179 33360936 12838952
10450908 87913121 48136920
86217924 26840528 56120044
45927526 57859648 71006340
78517906 1473320 78087118
71442912 552510 91721764
1886800 93725452 77764511
42965550 668045 152376
54791187 86850214 13790844
7524592 49683285 19399705
14321311 23848275 11091408
96124544 67376727 3644120
46866666 70903376 13102311
40286950 85055873 82191232
10326860 70677000 16468704
35997288 87136464 82498671
84454293 52717768 24357050
20863688 68477938 60161998
6387655 35367786 52832492
9307984 68479918 26853352
16456946 9016586 80886872
52131544 84046725 2659176
88109142 19762952 77689425
72907730 39004896 90689432
66151108 64564600 50435982
77019720 41086832 69761430
41694100 44524080 13181500
75129920 48893465 1475680
89705705 56968000 67830680
34895476 51065860 27493442
44963616 54338726 42897940
121800 85271327 12634664
33230995 41409216 78984504
19116016 44720224 52393154
49333945 86092626 13064768
45963479 75878274 30277124
63766872 79411984 45274704
61163730 91927750 41902960
62347471 11913385 17527939
15445063 30179448 30373888
6719691 18268128 39539036
19400262 93865002 44665872
90926450 25585555 46888542
97975317 53222992 47503160
57681709 57027698 54247050
62231976 47221422 56169056
45673032 4710762 96430843
34408558 95077865 44461882
35653940 42615372 68927864
76072721 13750778 27384240
40636700 82179738 50617242
50555680 13220112 42740216
23153064 55757988 82904791
16186716 39319344 54279565
97284958 33672240 45638420
17287000 3461150 88899938
7521234 42666285 69463456
81867660 40964460 47482161
53462034 28734926 35533161
81016776 92109615 69237067
50624286 82579216 94917019
11417702 24350416 29174431
35384293 85332741 59306175
27848666 44800090 38855460
46138864 41295798 3854546
24331072 44261405 59347207
73250318 83955744 25608210
75678726 28448084 186961
39766305 56436443 11764914
21281550 7671576 72716690
31850651 81787120 70064542
46727970 49031726 66003607
48417620 54542050 64102478
6681992 74955618 48582551
59012163 59627796 25322606
42727875 93918725 58081106
63884800 1206470 36088040
67393026 28227768 51748189
46471070 41801126 43056670
87153370 61302654 68909692
59969880 4279654 16334010
17337424 85909660 20274673
24564696 15042379 70917878
22651529 41420006 25043040
83796925 95054496 67446912
97939658 25470800 75461988
36623548 54277066 14119754
99602669 99826676 62772162
93182343 95885130 51189640
73257240 73123525 51218415
16653072 89417530 1912120
64528410 33360790 65471253
35889889 34742241 62447802
16277568 1131648 19065630
89798441 80712566 4148240
98572233 16128094 99862998
4911910 17411579 39796657
92128555 48698392 33611848
18864120 45299996 19062870
33397623 87413138 78679023
17061828 18091476 11148856
57948696 7267077 83460400
34266208 81400956 45916800
4009503 93938831 67064101
43285971 92042532 46105072
35433553 45046602 36444115
12198046 76881104 58690048
25638126 8652270 48624840
49130576 14218160 87794415
24054888 30152975 51959842
28902968 7500426 9318835
21870392 26618952 89242600
67236930 4498602 33886528
61662024 98543627 14534532
11513896 43916836 37306016
66433104 75466216 23284320
69436488 97008288 92496732
56049456 75995440 97434618
51680737 90835592 80111214
22142440 61047500 71631000
66629674 78019648 85116336
49292749 59119322 37122336
11057420 98103780 85422230
19461512 6146761 22989632
32983736 94324130 13922618
53316058 15471100 38630931
69144360 36799670 67707320
72566960 58444450 42961500
5235734 27664420 39305838
82522880 65049876 35584032
48452658 18470754 66744284
17773272 12067312 35220174
2161442 5067392 2798234
84332341 5636162 28471750
56100032 86995244 38101167
93259348 31016200 5320678
63211254 21333622 86648387
13010530 929450 53056568
31738000 78212480 3558882
32442100 35938917 68549350
2759252 75943670 6969730
74270300 77166757 25195752
8183626 35684481 64668620
96735200 31463270 66923370
18737625 75565872 45982664
11989148 69348386 15039418
5899315 89398110 64942152
20514080 11639460 20827557
24514312 78365336 98957580
84256442 16369287 87829244
4560470 79391676 58266240
25258315 59215280 55791024
53777168 52803460 5114790
26089500 20143548 9198849
88349486 67284351 48670370
57866160 97890037 41798015
40161920 65690263 71344776
51729016 55944535 1211574
77541146 54559709 89315416
51823174 94741780 22350316
33725945 32163250 6126526
16774273 11160610 52913380
6009516 91435596 19650943
17116182 65103853 79728390
66379247 16014456 8237888
35952878 65131488 85278975
63984000 72900515 97175488
46056226 89736340 35617872
5125280 35580214 64681334
29747200 20227478 13999886
56213620 89793746 65396160
1027208 44952019 35242830
93753635 32096358 2287652
12900670 7688313 1460502
2747920 59331375 97830975
56772408 70568244 62144355
22123562 80966504 17526266
70204030 11546180 2399787
23830900 97882075 72177504
86480295 34631376 90773311
47966062 69223716 62130717
17279302 4822934 5404896
93444464 32250625 78088545
52082834 1444164 30482250
58885536 98309230 35600992
15638776 12787773 97737120
7327572 18935840 56685621
62180312 32837280 85062765
9789642 73967128 10555895
65921920 90738945 95715000
42407966 41812433 38207445
96476904 16705200 11089440
41124805 88522082 38094593
12223808 2494872 55427625
42103208 41131800 24169692
69294282 64695100 24284429
13882240 24931448 57717984
57368245 12148480 16437383
47499135 17518875 65148304
1848972 80310326 4429260
55096782 41186215 65830607
99820044 7581020 31201840
21817834 81900897 20032007
29530878 30111974 8663160
88236696 95383360 22615065
28011830 96690000 62503822
67564780 68753031 647813
46002492 53297462 52213380
79361910 73366463 87442720
4410294 10757716 35370813
13239010 2844250 4492720
92422357 22427343 38983340
48642983 47698040 21015001
65803352 46531341 1580320
34768552 69246939 2532775
10033028 11953254 92613580
5265152 75424344 18918000
71468529 45179169 75986074
25403096 34259624 65729265
48472040 38071658 66326829
64683700 80982998 37588000
41649197 79438371 85518048
64646488 83452750 46130547
92748306 95228158 2543310
45893477 13366104 9577504
9119316 26602022 74641472
93936366 29032414 19762500
92150875 67895800 40234460
22402176 92447000 47620582
37139785 78832152 58363586
89677644 51887229 67158019
69901776 73647872 23056314
57488696 21802148 25894394
5869980 30220560 76269700
15896352 46475720 35031292
26087168 83810560 8433478
18995056 8631104 21527984
23495860 24930378 37689214
85112220 90634026 454412
21352707 2855776 34468100
32575096 10859312 28313655
50641776 58989185 39033850
84816403 17062450 4462968
41781509 53555125 17571873
47661824 32375802 38579165
44235808 26623446 26798735
14802451 92748495 21285490
49484852 7859790 92377090
70628419 79567765 55126336
24789400 37727639 30159748
67614068 62729336 59083287
78569450 43701142 34349304
90724752 9592420 30000438
85306444 82704040 77627648
97263328 19307912 51051894
3495700 57088676 96435292
16053323 71247852 54917420
43039579 18582 1237253
38667210 69485680 50641743
90816500 32642400 51771742
67434720 34427160 96674548
16906580 49256980 71650848
39977880 53015863 85367344
77588710 47842432 50160993
22739280 52750152 70019635
94996500 29943000 84238776
58698534 22273344 71480008
66553384 30641354 85448225
17133742 39800693 31281344
12750264 76970988 97999008
63983476 23842600 29431152
23251884 49493416 68989733
22626240 98344601 92820900
44748864 52801714 98768544
44140085 12245280 75368560
15263790 1027430 19096605
82381792 5668518 31665543
67166900 62197904 50481504
67907132 20918406 14027148
40231166 71999230 55876608
39214576 52323500 55330560
98254208 91080911 97308926
58042330 35968310 58401147
31703545 96148822 23394220
74255400 58410108 34593470
75578625 37899237 41071944
81405024 33018325 1100550
35586064 69758000 56650680
75829100 54186716 41881620
97587918 14868920 53201180
73937080 54890698 2236961
68720800 58184544 84771456
10570098 46413576 45896109
54990310 49361863 37710260
53908096 5956830 35135880
80625210 22323892 23755508
4112208 4638136 557975
23310450 9810632 76633100
41435162 62035588 8688944
83729030 80224032 51042962
41063310 23323200 10165050
84333544 44477102 80596800
26002231 18410652 29869599
1276425 92355584 70034342
34135585 6986615 12267756
25197906 36526080 8738876
63625457 55647750 4488988
96572784 29436978 5205500
32533274 79091456 24146147
64174100 84187136 46014654
99425812 21729560 98833479
752951 22594357 53544020
54077455 13610320 77370498
63529084 73962146 7922585
24792310 63210425 27646080
84182100 92011544 86335120
57922468 16570580 75411655

