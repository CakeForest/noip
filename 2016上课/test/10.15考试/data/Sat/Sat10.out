2883147121585
